src/cpp/server/db/README.md

```diff
@@ -0,0 +1,23 @@
+## Server Database Schemas
+
+This folder contains the database schemas for the RStudio Server database. These schemas are copied to the build output directory, and are applied to the database every time `rserver` is started.
+
+Schema files rely on a file format convention containing the following:
+* The date time at which the script was generated in the format YYYYMMDDHHmmssuuu (where uuu is microseconds)
```

Comment:
No, but it absolutely ensures no duplicates. Also, I picked a method that was easy to script, so that plays a big part into what I chose to do (see `./make-schema.sh`